### PR TITLE
Email tests

### PR DESCRIPTION
**Pull request by andersonresende**
- Emails tests
- Database tests
- Fixed deprecated warnings

### CODE REVIEW

File: fastapi_backend/app/routes/items.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ async def read_item(
 ):
     result = await db.execute(select(Item).filter(Item.user_id == user.id))
     items = result.scalars().all()
-    return [ItemRead.from_orm(item) for item in items]
+    return [ItemRead.model_validate(item) for item in items]
```
Comment (fjsj) [inline fastapi_backend/app/routes/items.py:22]: Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

File: fastapi_backend/app/schemas.py [hunk 2]
```diff
@@ -31,6 +31,4 @@ class ItemRead(ItemBase):
     id: UUID
     user_id: UUID
 
-    class Config:
-        orm_mode = True
-        from_attributes = True
+    model_config = {"from_attributes": True}
```
Comment (fjsj) [inline fastapi_backend/app/schemas.py:34]: Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

File: fastapi_backend/tests/conftest.py [hunk 3]
```diff
@@ -58,7 +58,9 @@ async def override_get_async_session():
     app.dependency_overrides[get_user_db] = override_get_db
     app.dependency_overrides[get_async_session] = override_get_async_session
 
-    async with AsyncClient(app=app, base_url="http://localhost:8000") as client:
+    async with AsyncClient(
+        transport=ASGITransport(app=app), base_url="http://localhost:8000"
+    ) as client:
```
Comment (fjsj) [inline fastapi_backend/tests/conftest.py:63]: Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

File: fastapi_backend/tests/test_email.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    assert config.MAIL_USERNAME == "test_user"
+    assert config.MAIL_PASSWORD == "test_pass"
+    assert config.MAIL_FROM == "test@example.com"
+    assert config.MAIL_PORT == 587
+    assert config.MAIL_SERVER == "smtp.test.com"
+    assert config.MAIL_FROM_NAME == "Test Sender"
+    assert config.MAIL_STARTTLS is True
+    assert config.MAIL_SSL_TLS is False
+    assert config.USE_CREDENTIALS is True
+    assert config.VALIDATE_CERTS is True
```
Comment (fjsj) [inline fastapi_backend/tests/test_email.py:47]: use assert and assert not instead of is True and is False.
`is` should be used only for object identity checks.

File: fastapi_backend/app/schemas.py (hunk 2 above)
Comment (andersonresende) [reply fastapi_backend/app/schemas.py:34]: It was deprecated, I got the warning and updated it.

File: fastapi_backend/tests/conftest.py (hunk 3 above)
Comment (andersonresende) [reply fastapi_backend/tests/conftest.py:63]: It was deprecated, I got the warning and updated it.

File: fastapi_backend/tests/test_email.py (hunk 4 above)
Comment (andersonresende) [reply fastapi_backend/tests/test_email.py:47]: makes sense.

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall LGTM. Please check comments.

File: fastapi_backend/app/routes/items.py (hunk 1 above)
Comment (andersonresende) [reply fastapi_backend/app/routes/items.py:22]: It was deprecated, I got the warning and updated it.

**Review by fjsj — APPROVED** [review]
